lib/ansible/modules/system/aix_lvol.py

```diff
@@ -0,0 +1,354 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2016, Alain Dejoux ²adejoux@djouxtech.net>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'version': '1.0'}
+
+DOCUMENTATION = '''
+---
+author:
+    - "Alain Dejoux (@adejoux)"
+module: aix_lvol
+short_description: Configure AIX LVM logical volumes
+description:
+  - This module creates, removes or resizes AIX logical volumes. Inspired by lvol module.
+version_added: "2.3"
+options:
+  vg:
+    description:
+    - The volume group this logical volume is part of.
+    required: true
+  lv:
+    description:
+    - The name of the logical volume.
+    required: true
+  lv_type:
+    description:
+    - The type of the logical volume. Default to jfs2.
+    required: false
```

Comment:
If parameters are not required, you do not have to add `required: false`. This is the implied value, so it is easier to see what is required if only the required parameters have this option.